osu.Game.Rulesets.Osu.Tests/TestSceneTouchInput.cs

```diff
@@ -116,9 +165,224 @@ public void TestPositionalInputUpdatesOnlyFromMostRecentTouch()
             endTouch(TouchSource.Touch2);
             checkPosition(TouchSource.Touch2);
 
-            // note that touch1 was never ended, but becomes active for tracking again.
+            // note that touch1 was never ended, but is no longer valid for touch input due to touch 2 occurring.
```

Comment:
This is a minor change to existing behaviour, but I think it's actually better than what we had (stops any kind of positional feedback when more than one finger is on screen and moving at once).